app/src/main/java/se/inera/intyg/minaintyg/error/service/LogErrorService.java

```diff
@@ -0,0 +1,29 @@
+package se.inera.intyg.minaintyg.error.service;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import se.inera.intyg.minaintyg.error.service.dto.LogErrorRequest;
+import se.inera.intyg.minaintyg.logging.MonitoringLogService;
+import se.inera.intyg.minaintyg.user.UserService;
+
+@RequiredArgsConstructor
+@Service
+public class LogErrorService {
+
+  private final MonitoringLogService monitoringLogService;
+  private final UserService userService;
+
+  public void log(LogErrorRequest request) {
+    if (userService.getLoggedInUser().isEmpty()) {
```

Comment:
This is redundant as the endpoint gets the same security as other endpoints (requires authenticated session).